Vokal-Swift.xctemplate/UserAPITests.swift

```diff
@@ -42,31 +42,31 @@ class UserAPITests: XCTestCase {
         let expectation = self.expectation(description: "Successful login")
         
         //WHEN: User logs in with valid credentials
-        UserAPI.loginWithEmail(ValidLoginEmail,
-            password: ValidLoginPassword,
-            success: {
-                resultDict in
-                //THEN: The call should succeed and the token should match the mock token.
-                if let token = resultDict["token"] as? String {
-                    XCTAssertEqual(token, self.MockLoginToken)
-                } else {
-                    XCTFail("No token returned!")
-                }
-                
-                //THEN: The token should already be stored in the keychain.
-                if let keychainToken = TokenStorageHelper.getAuthorizationToken() {
-                    XCTAssertEqual(keychainToken, self.MockLoginToken)
-                } else {
-                    XCTFail("Token was not stored in the keychain!")
-                }
-                
-                expectation.fulfill()
-            },
-            failure: {
-                error in
-                XCTFail("Failure block hit with error \(error)")
-                
-                expectation.fulfill()
+        UserAPI.login(with: ValidLoginEmail,
```

Comment:
I think the declaration was updated so the call site here needs to be updated with `login(withEmail:`. Ditto for line 82 and 111 in this file.